app/support/order_details/cost.rb

```diff
@@ -0,0 +1,20 @@
+class OrderDetails::Cost
+  include ActionView::Helpers::NumberHelper
+
+  def initialize(order_details)
+    @order_details = order_details
+  end
+
+  def final_total
```

Comment:
I need to look a little more closely (since it's some metaprogramming), but I think `Order` actually has a total method already. I think it might be better to wrap that, or simply add the `number_to_currency(@order.total)` to the view without the need for this class.